controllers/tokenupload_controller.go

```diff
@@ -108,8 +108,6 @@ func (r *TokenUploadReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	secretType := uploadSecret.GetLabels()[tokenSecretLabel]
 	switch secretType {
```

Comment:
Do you really need this switch at all?
I guess the predicate below should do this job?